dockerfiles/README.md

```diff
@@ -90,7 +90,9 @@ docker build --build-arg package_url=https://storage.openvinotoolkit.org/reposit
 ----------------
 
 * Custom image with CPU, iGPU, VPU support  
-To learn more, follow [Build custom Intel® Distribution of OpenVINO™ toolkit Docker image](ubuntu18/build_custom/README.md).
+We prepared Dockerfiles for custom development OpenVINO™ from source. To learn more, follow:
```

Comment:
```suggestion
Use Dockerfiles to develop a custom version of OpenVINO™ from source. To learn more, follow:
```

Reply:
May be replace it to something like this:

You can use Dockerfiles from `build_custom` folders to build a custom version of OpenVINO™ from source codes for development